code/modules/organs/organ_external.dm

```diff
@@ -137,53 +180,32 @@
 				burn = max(0, burn - can_inflict)
 		//If there are still hurties to dispense
 		if (burn || brute)
-			if (status & ORGAN_ROBOT && body_part != UPPER_TORSO && body_part != LOWER_TORSO)
-				droplimb(1) //Robot limbs just kinda fail at full damage.
-			else
-				//List organs we can pass it to
-				var/list/datum/organ/external/possible_points = list()
-				if(parent)
-					possible_points += parent
-				if(children)
-					possible_points += children
-				if(forbidden_limbs.len)
-					possible_points -= forbidden_limbs
-				if(possible_points.len)
-					//And pass the pain around
-					var/datum/organ/external/target = pick(possible_points)
-					target.take_damage(brute, burn, sharp, edge, used_weapon, forbidden_limbs + src)
+			owner.shock_stage += brute+burn
```

Comment:
Let damage transfer up to the parent organ; There was a bug that lingered around for a few weeks where IPC's damage wouldn't transfer up to the parent organ--it confused players to no end why the "immortal IPC" wouldn't die; this will generate the same confusion only for everyone.